packages/server/src/api/controllers/row/views.ts

```diff
@@ -27,58 +22,23 @@ export async function searchView(
 
   const { body } = ctx.request
 
-  // Enrich saved query with ephemeral query params.
-  // We prevent searching on any fields that are saved as part of the query, as
-  // that could let users find rows they should not be allowed to access.
-  let query = dataFilters.buildQuery(view.query || [])
```

Comment:
I believe this is needed to handle if the `view.query` is of type `SearchFilter[]` not `SearchFilters`, which will be the case in the DB, so we need to keep a call to `buildQuery.`

Reply:
Yes, I moved it in the sdk, otherwise the SDK and the controller would need to be highly coupled